### PR TITLE
Add classifiers to setup.py

### LINKED ISSUE #29: Bump version for py3 support and add classifiers
Here's some art:

https://github.com/pre-commit/pre-commit/blob/master/setup.py#L26

### PR DESCRIPTION
**Pull request by dinahshi**
Closes #29 

Unclear why py26 is failing on missing test coverage for a test file, thought I'd open the PR and get some input: @asottile @tomelm @mittonk

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,6 +1,8 @@
 language: python
 env: # These should match the tox env list
+    - TOXENV=py26
     - TOXENV=py27
+    - TOXENV=py33
```
Comment (asottile) [inline .travis.yml]: I wouldn't bother with py33 or py26 unless you want to put effort into supporting them.  They're both end-of-lifed and anybody sane is going to be using py27, py34, or py35

The coverage issue in py26 is due to a coveragepy bug:

https://bitbucket.org/ned/coveragepy/issues/146/context-managers-confuse-branch-coverage

File: tox.ini [hunk 2]
```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py27,py34
+envlist = py26,py27,py33,py34
```
Comment (asottile) [inline tox.ini]: maybe add pypy, pypy3 too?

Then you can add 

```
'Programming Language :: Python :: Implementation :: PyPy',
```

to classifiers as well :D

**Comment by asottile** [comment]
fixnship

**Comment by dinahshi** [comment]
Looks like pypy is [not happy](https://travis-ci.org/Yelp/yelp-python/jobs/94538000) about the dynamically added methods on client (from #31). How badly do we want to support pypy?

We could change the tests to reference the endpoint classes but I think calling them off the client gives us extra assurance that they're being copied to the client successfully.

**Comment by asottile** [comment]
Wow, usually this means that you're doing something way too magical :S

I guess don't support pypy unless you want to change the code to be more reasonable

**Comment by asottile** [comment]
Yeah looking at that code, inspect is usually not a good idea especially for maintainability :S

**Comment by dinahshi** [comment]
We can replace that `inspect.getmethods` call with a `dir` and manually filter for methods but I don't think that would change the pypy error as the Client class would still be populated dynamically.

**Comment by mittonk** [comment]
Ah, the fun of compiler-specific features.  

Short term: Yep, if our stuff doesn't work on PyPy we shouldn't claim it does.

Medium term: This introspection may indeed be a bit too fancy.  @dinahshi , want to open an Issue for a simpler implementation?  We go for PyPy support at that point.

**Comment by mittonk** [comment]
:shipit:

**Comment by asottile** [comment]
You can probably still have magic, but I'd suggest calling type() to build
the classes with dynamic methods instead of attaching methods after the
class is made.

**Pull request merged**